mmv1/products/compute/Subnetwork.yaml

```diff
@@ -73,6 +73,9 @@ iam_policy:
 custom_code:
   extra_schema_entry: 'templates/terraform/extra_schema_entry/subnetwork.tmpl'
   constants: 'templates/terraform/constants/subnetwork.tmpl'
+  decoder: 'templates/terraform/decoders/compute_subnetwork.go.tmpl'
+  encoder: 'templates/terraform/encoders/compute_subnetwork.go.tmpl'
+  update_encoder: 'templates/terraform/update_encoder/compute_subnetwork.go.tmpl'
```

Comment:
these are all identical to the ones for `compute_network`; I'm assuming there's still a preference to have these be copied / pasted vs. sharing a common file with a more generic name?